md/rest-api-overview.md

```diff
@@ -53,6 +59,99 @@ To log out, use the following request:
 
 Setting the redirect parameter to false indicates that the service should not redirect to the login page after logging out.
 
+### Walk-through: Example shows how to instantiate one case using the REST API
```

Comment:
Walk-through: How to start a case uning the REST API
(anyway, "Example shows" can be removed)